easybuild/easyconfigs/l/Lua/Lua-5.3.4-GCCcore-6.3.0.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = "Lua"
+version = "5.3.4"
+
+homepage = "http://www.lua.org/"
+description = """Lua is a powerful, fast, lightweight, embeddable scripting language.
+ Lua combines simple procedural syntax with powerful data description constructs based
+ on associative arrays and extensible semantics. Lua is dynamically typed,
+ runs by interpreting bytecode for a register-based virtual machine,
+ and has automatic memory management with incremental garbage collection,
+ making it ideal for configuration, scripting, and rapid prototyping."""
+
+toolchain = {'name': 'GCCcore', 'version': '6.3.0'}
+
+source_urls = ['http://lua.org/ftp/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+
+# Standard install of lua does not have a configure step, thus skipping the configure step:
+
+skipsteps = ['configure']
+
+buildopts = "linux"
+
+installopts = "INSTALL_TOP=%(installdir)s"
+
+dependencies = [
+	     ('ncurses', '6.0'),
```

Comment:
from style check: indentation contains mixed spaces and tabs, continuation line over-indented for hanging indent